Sources/SwiftInspectorVisitors/Tests/ProtocolVisitorSpec.swift

```diff
@@ -191,6 +191,23 @@ final class ProtocolVisitorSpec: QuickSpec {
               expect(protocolInfo?.innerTypealiases.first?.initializer?.asSource) == "Any"
             }
           }
+
+          context("with properties") {
+            it("finds the typealias") {
+              let content = """
+                public protocol SomeProtocol {
+                  var foo: Int { get set }
+                  var bar: Double { get }
+                }
+                """
+
+              try self.sut.walkContent(content)
+
+              let protocolInfo = self.sut.protocolInfo
+              expect(protocolInfo?.properties.first?.name) == "foo"
+              expect(protocolInfo?.properties.last?.name) == "bar"
```

Comment:
can we add tests that we get the type as well?

Reply:
@dfed these tests are already in the `PropertySyntaxVisitorSpec`. I don't think it'd be useful to have duplicated tests here since then we'd be double testing the `PropertySyntaxVisitor` making refactoring it harder.

Reply:
alright I buy that. I've erred towards more integration tests rather than fewer and I've found that to be helpful. But in this case there's less nesting potential so we're likely fine.